collector/logical_router_collector.go

```diff
@@ -67,14 +84,40 @@ func (c *logicalRouterCollector) Collect(ch chan<- prometheus.Metric) {
 		level.Error(c.logger).Log("msg", "Unable to list logical routers", "err", err)
 		return
 	}
+	logicalRouterStatusMetrics := c.generateLogicalRouterStatusMetrics(logicalRouters)
+	for _, lrouterMetric := range logicalRouterStatusMetrics {
+		labels := []string{lrouterMetric.ID, lrouterMetric.Name, lrouterMetric.TransportNodeID, lrouterMetric.ServiceRouterID, lrouterMetric.HighAvailabilityStatus}
+		ch <- prometheus.MustNewConstMetric(c.logicalRouterStatus, prometheus.GaugeValue, 1.0, labels...)
```

Comment:
Shouldn't we include the value of all possible status?

Reply:
fixed in  142421d